build-system/server.js

```diff
@@ -56,9 +56,9 @@ setInterval(function() {
   }
 }, 1000);
 
-const middleware = [];
+const middleware = [app];
```

Comment:
You verified that it doesn't matter if `app` is first?

Reply:
In the sense that this is how it was before the conditional on `quiet` was added ;) (and that `gulp pr-check` passes for me locally)

Reply:
Changed it to be after `morgan`